assets/bulk-add.js

```diff
@@ -0,0 +1,37 @@
+class BulkAdd extends HTMLElement {
+  constructor() {
+    super();
+    this.queue = []
+    this.requestStarted = false;
+  }
+
+  startQueue(id, quantity) {
+    this.queue.push({id, quantity})
+    const int = setInterval(() => {
+      if (this.queue.length > 0) {
+        if (!this.requestStarted)  {
+          this.sendRequest(this.queue)
+        }
+      } else {
+        clearInterval(int)
+      }
+    }, 250)
+  }
+
+
+  sendRequest(queue) {
+    this.requestStarted = true;
+    const items = {}
+    const ids = []
+    queue.forEach((q) => {
+      items[parseInt(q.id)] = q.quantity;
+    });
+    queue.forEach((q) => {
+      ids.push(q.id)
+    })
+    this.queue = this.queue.filter(q => !queue.includes(q));
+    this.updateMultipleQty(items, ids)
```

Comment:
This is a little scary—that function isn't defined in this class, so there's no guarantee it will exist if this class is re-used in a different context. Does it need to be somewhere else, or could it move into this class?

Reply:
So I wanted to move this to this class but because I havent done the refactoring for quick add bulk yet, i didnt want to have it on this PR yet 🤔 

I thought it was worth it to have this live in a reusable component given I will reuse this in quick add bulk

Also not all functions will be exactly the same for QOL and quick add bulk, but I figured I could call them the same - or do you think this would still feel a bit risky?

Reply:
ahhh that makes sense. And yeah, no problem with having them same name for both (because I was raised on Object Orientation, I call it [Polymorphism](https://stackify.com/oop-concept-polymorphism/) and consider it a good thing)

Reply:
And fine to leave it as-is for now 👍